mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/mapping/impl/PojoMappingDelegateImpl.java

```diff
@@ -142,4 +145,19 @@ public PojoIndexer createIndexer(PojoWorkSessionContext context) {
 				context
 		);
 	}
+
+	@Override
+	public ConfiguredSearchIndexingPlanFilter configuredSearchIndexingPlanFilter(SearchIndexingPlanFilter filter) {
+		SearchIndexingPlanFilterContextImpl context = new SearchIndexingPlanFilterContextImpl( typeManagers );
+		filter.apply( context );
+		return context.createFilter();
+	}
+
+	@Override
+	public ConfiguredSearchIndexingPlanFilter configuredSearchIndexingPlanFilter(SearchIndexingPlanFilter filter,
+			ConfiguredSearchIndexingPlanFilter fallback) {
+		SearchIndexingPlanFilterContextImpl context = new SearchIndexingPlanFilterContextImpl( typeManagers );
+		filter.apply( context );
+		return context.createFilter( fallback );
+	}
```

Comment:
If you just expose the second method and pass `null` where relevant, you can get rid of the `*Creator` interface and expose the method directly on `HibernateOrmSearchSessionMappingContext`.